From 39199396f31dbb196e25f5ac0f9ca89749d379a7 Mon Sep 17 00:00:00 2001
From: Bogdan107 <35569459+Bogdan107@users.noreply.github.com>
Date: Sun, 5 Dec 2021 17:50:01 +0000
Subject: [PATCH] Update Syntax.in - fix colorize *.zsh files

I need colorizing content of the "*.zsh" files in mcedit.
With "zsh(env|rc)" regexp rule, only "*.zshenv" and "*.zshrc" files are will be colorized.
With "zsh(env|rc)?" regexp rule, "*.zsh" files will be colorized too.
---
 misc/syntax/Syntax.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/syntax/Syntax.in b/misc/syntax/Syntax.in
index eef0895425..07905d09e0 100644
--- a/misc/syntax/Syntax.in
+++ b/misc/syntax/Syntax.in
@@ -37,7 +37,7 @@ include diff.syntax
 file ..\*\\.lsm$ LSM\sFile
 include lsm.syntax
 
-file .\*(bash(_completion|rc)|profile|\\.(sh|bash(rc|_(completion|profile|log(in|out)|aliases|exports|history))|profile|zlog(in|out)|zprofile|zsh(env|rc)))$ Shell\sScript ^#!\s\*/(.\*/|usr/bin/env\s)([a-z]?|ba|pdk)sh
+file .\*(bash(_completion|rc)|profile|\\.(sh|bash(rc|_(completion|profile|log(in|out)|aliases|exports|history))|profile|zlog(in|out)|zprofile|zsh(env|rc)?))$ Shell\sScript ^#!\s\*/(.\*/|usr/bin/env\s)([a-z]?|ba|pdk)sh
 include sh.syntax
 
 # Since openrc scripts have no extension "by definition", use an empty pattern
